lib/src/mapbox_maps_platform.dart

```diff
@@ -6,11 +6,18 @@ class _MapboxMapsPlatform {
   late final MethodChannel _channel = MethodChannel(
```

Comment:
We have to add suffix to the name of this channel otherwise all kinds of events could end up broadcasting across all active map instances.